hotel-bookings-api/server.js

```diff
@@ -15,6 +16,66 @@ app.get("/", function (request, response) {
 
 // TODO add your routes and helper functions here
 
-const listener = app.listen(process.env.PORT, function () {
+// all the bookings
+app.get("/bookings", function (request, response) {
+  response.send(bookings);
```

Comment:
As a developer it's always a good idea to introduce extra layers of securities (whenever we can of course) to make sure our program does not crash all of a sudden. A good example of that here would be adding an extra check before we send a booking, when the GET request is called. So on line 21, you can do the following:
`res.status(200).send(bookings);`

This means that if the GET request has been successful (CODE 200) is an indication of that, only after will it send the entire bookings. 

You can read more on Status Codes on the excellent MDN - [link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)